packages/simcore-sdk/tests/integration/conftest.py

```diff
@@ -323,3 +325,26 @@ def _assign_config(
         )
         if not entry[2] is None:
             config_dict[port_type].update({entry[0]: entry[2]})
+
+
+@pytest.fixture
+async def r_clone_settings(
+    minio_config: Dict[str, Any], storage_service: URL
+) -> RCloneSettings:
+    client = minio_config["client"]
+    settings = RCloneSettings.parse_obj(
+        dict(
+            R_CLONE_S3=dict(
+                S3_ENDPOINT=client["endpoint"],
+                S3_ACCESS_KEY=client["access_key"],
+                S3_SECRET_KEY=client["secret_key"],
+                S3_BUCKET_NAME=minio_config["bucket_name"],
+                S3_SECURE=client["secure"],
+            ),
+            R_CLONE_PROVIDER=S3Provider.MINIO,
+        )
+    )
+    if not await is_r_clone_available(settings):
+        pytest.skip("rclone not installed")

```

Comment:
Q: will this skip the whole test? is it not the purpose of your system that it should still work even when RClone is not available?

Reply:
Had another look at it. Did some refactoring, and now it will work as expected.
Only tests which absolutely require `rclone` will be skipped, if it is missing.